lib/shopify_api/rest/request.ex

```diff
@@ -72,15 +75,43 @@ defmodule ShopifyAPI.REST.Request do
 
   ## Private Helpers
 
-  defp log_request(method, url, time, response) do
+  defp send_telemetry(
+         %{app_name: app, shop_name: shop},
+         method,
+         url,
+         time,
+         {:ok, %{status_code: status}} = response
```

Comment:
It may also be valuable to generate telemetry events for requests that fail? (e.g. for reasons of timeouts, etc).